src/unity/CMakeLists.txt

```diff
@@ -71,20 +71,20 @@ endif()
 ############################################################
 
 if (WIN32 OR is_xcode)
-  group_sources(src)
-  group_sources(extras/rippled/src)
-  group_sources(extras/rippled/Builds)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" src)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" extras/rippled/src)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" extras/rippled/Builds)
 endif()
 
 ############################################################
 
 add_with_props(lib_src ripple-libpp.cpp
-  -I"${CMAKE_SOURCE_DIR}/"extras/rippled/src/secp256k1
+  -I"${CMAKE_CURRENT_SOURCE_DIR}/"../../extras/rippled/src/secp256k1
```

Comment:
yeah, it works fine for me. I'm willing to bet that the quotes got there in the first place because someone had a space somewhere in their checkout path. I just did a quick test and it unfortunately does fail if you checkout into any path that has a space.  Adding quotes around the entire include path (e.g. `-I"${CMAKE_CURRENT_SOURCE_DIR}/../../extras/rippled/src/secp256k1"`) works fine though. Sorry for the confusion.